appcoins-ads/build.gradle

```diff
@@ -25,19 +27,25 @@ android {
   }
 
   buildTypes {
+    debug {
+      minifyEnabled false
+      buildConfigField 'String', 'BDS_WALLET_PACKAGE_NAME',
+          project.BDS_WALLET_PACKAGE_NAME_DEV
+    }
     release {
+      buildConfigField 'String', 'BDS_WALLET_PACKAGE_NAME',
+          project.BDS_WALLET_PACKAGE_NAME
       minifyEnabled false
     }
-    debug {}
   }
 
   lintOptions {
     warning 'InvalidPackage'
   }
 
   compileOptions {
-    targetCompatibility TARGET_COMPATIBILITY
-    sourceCompatibility SOURCE_COMPATIBILITY
+    targetCompatibility TARGET_COMPATIBILITY_ADS
```

Comment:
Since we are no longer using lambdas, this can be removed

Reply:
Removed